File: leetcode/961. N-Repeated Element in Size 2N Array/961.js
// https://helloacm.com/how-to-find-n-repeated-element-in-size-2n-array/ 
// https://leetcode.com/problems/n-repeated-element-in-size-2n-array/

/**
 * @param {number[]} A
 * @return {number}
 */
var repeatedNTimes = function(A) {
    let hash = {};
    for (let x of A) {
        if (hash[x]) {
            return x;
        }
        hash[x] = true;
    }
};

File: leetcode/961. N-Repeated Element in Size 2N Array/961.py
# https://helloacm.com/how-to-find-n-repeated-element-in-size-2n-array/ 
# https://leetcode.com/problems/n-repeated-element-in-size-2n-array/

class Solution:
    def repeatedNTimes(self, A):
        """
        :type A: List[int]
        :rtype: int
        """
        hash = {}
        for i in A:
            if i in hash:
                return i
            hash[i] = True